docs/import/create_score_asset.js

```diff
@@ -49,7 +39,41 @@ function countDamageAndBuildings(feature, damage, buildings) {
   } else {
     properties = properties.set(damageTag, 0);
   }
-  return ee.Feature(geometry, properties);
+  const snapPop = ee.Number(feature.get(snapPopTag)).long();
```

Comment:
This works if feature.get(snapPopTag) is null? I would expect it to throw an error.

Reply:
Didn't see an algs 'or' function. Figure this works? If either is null we have no real shot at getting a percentage.